twitter4j-example.yml

```diff
@@ -0,0 +1,5 @@
+# Twitter4j properties
```

Comment:
suggestion would be to nest this in a Twitter node level object.  The oauth prefix could be optional as well.